### PR TITLE
[enterprise-4.11] OSDOCS-3743: RN entry for kdump

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OSDOCS-3743

Link to docs preview:
- RHCOS new features: http://file.rdu.redhat.com/jdohmann/OSDOCS-3731-RN/release_notes/ocp-4-11-release-notes.html#ocp-4-11-rhcos
- Technology Preview table: http://file.rdu.redhat.com/jdohmann/OSDOCS-3731-RN/release_notes/ocp-4-11-release-notes.html#ocp-4-11-technology-preview

Additional information: RN entry for #46629

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -71,6 +71,16 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 
 {product-title} 4.11 introduces the `nvme-cli` package that provides an interface for communicating with NVMe disks using host software capabilities.
```
Comment (miabbott) [inline release_notes/ocp-4-11-release-notes.adoc:73]: This is a bit misleading; RHCOS has had support for using NVMe disks for a while.  The `nvme-cli` package is just a tool that can be used to manage NVMe devices.

From `rpm -qi nvme-cli`:

```
Summary     : NVMe management command line interface
Description : nvme-cli provides NVM-Express user space tooling for Linux.
```

**Comment by miabbott** [comment]
Bit of a correction for `nvme-cli`, but otherwise looks good

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (jldohmann) [reply release_notes/ocp-4-11-release-notes.adoc]: Thanks for catching this, fixed

**Review by miabbott — APPROVED** [review]

**Review by kelbrown20 — COMMENTED** [review]
Looks great, LGTM!

**Review by kalexand-rh — COMMENTED** [review]
I see a markup tweak that should be fixed before it's merged and have a suggestion.

**Comment by jldohmann** [comment]
23a1f14..432ab4d implement more peer review feedback

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -67,9 +67,19 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
-==== Support for NVMe over Fabrics
+==== Improved support for NVMe over Fabrics
 
-{product-title} 4.11 introduces the `nvme-cli` package that provides an interface for communicating with NVMe disks using host software capabilities.
+{product-title} 4.11 introduces the `nvme-cli` package that provides an interface for managing NVMe devices.
+
+[id="ocp-4-11-kdump-x86"]
+==== Investigate kernel crashes on AMD64 machines with kdump
+
+{op-system} now supports kdump for the `x86_64` architecture in {product-title} 4.11. Support for kdump on other architectures remains in Technology Preview.
```
[the remark this answers is not in the document]
Comment (jldohmann) [reply release_notes/ocp-4-11-release-notes.adoc]: thanks, fixed

File: release_notes/ocp-4-11-release-notes.adoc [hunk 3]
```diff
@@ -729,10 +739,25 @@ In the table below, features are marked with the following statuses:
 |GA
 |GA
 
-|Kdump
+|Kdump (`x86_64` architecture)
```
[the remark this answers is not in the document]
Comment (jldohmann) [reply release_notes/ocp-4-11-release-notes.adoc]: thanks, added

**Pull request merged**